booking/src/main/java/org/dcsa/conformance/standards/booking/party/BookingCarrier.java

```diff
@@ -301,8 +303,10 @@ private void requestToAmendConfirmedBooking(JsonNode actionPrompt) {
     Consumer<ObjectNode> bookingMutator =
         booking ->
             booking
-                .putArray("requestedChanges")
+                .putArray("feedbacks")
                 .addObject()
+                .put("severity", "WARN")
+                .put("code", "PROPERTY_VALUE_HAS_BEEN_CHANGED")
                 .put(
```

Comment:
I would expect a Request for amendment to a confirmed bookigto use `ERROR` and `PROPERTY_VALUE_MUST_CHANGE`